x-pack/plugins/maps/server/kibana_server_services.ts

```diff
@@ -5,7 +5,10 @@
  * 2.0.
  */
 
-import { ISavedObjectsRepository } from 'kibana/server';
+import { ElasticsearchClient, ISavedObjectsRepository } from 'kibana/server';
+import { IndexPatternsService } from '../../../../src/plugins/data/server';
```

Comment:
I think referencing `IndexPatternsCommonService` would be less disruptive than renaming `IndexPatternsService` and `IndexPatternsServiceProvider`

Reply:
Hey Matt! Need some help with this one. It looks like `IndexPatternsCommonService` is an unexported rename from the data plugin located [here](https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/src/plugins/data/server/index_patterns/index_patterns_service.ts#L22). Assuming I've got the right one, how does that help us here?

> would be less disruptive than renaming IndexPatternsService and IndexPatternsServiceProvider

Where are you seeing these renames?

Reply:
Following some offline conversation, I've exported the alias from the data plugin and updated local refs. There should be some follow-up work on the data side to update any relevant refs to point at this alias since that's beyond the scope of this PR